cli/tests/unit/globals_test.ts

```diff
@@ -58,37 +60,51 @@ unitTest(function DenoNamespaceImmutable(): void {
   try {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     (Deno as any) = 1;
-  } catch {}
+  } catch {
+    noop();
```

Comment:
Chose to use a `noop` function as opposed to ignoring the empty catch block for explicitness's sake.